etc/schema/mysql-upgrade/v1.2.0.sql

```diff
@@ -17,9 +17,11 @@ ALTER TABLE x509_certificate
     CHANGE ca ca enum('n', 'y') NOT NULL,
     CHANGE trusted trusted enum('n', 'y') NOT NULL DEFAULT 'n';
 
-ALTER TABLE x509_certificate_chain CHANGE valid valid enum('n', 'y', 'yes', 'no') NOT NULL DEFAULT 'n';
+ALTER TABLE x509_certificate_chain
+    CHANGE valid valid enum('n', 'y', 'yes', 'no') NOT NULL DEFAULT 'n',
+    CHANGE valid valid enum('n', 'y') NOT NULL DEFAULT 'n';
 
 UPDATE x509_certificate_chain SET valid = 'y' WHERE valid = 'yes';
 UPDATE x509_certificate_chain SET valid = 'n' WHERE valid = 'no';
 
-ALTER TABLE x509_certificate_chain CHANGE valid valid enum('n', 'y') NOT NULL DEFAULT 'n';
+ALTER TABLE x509_target ADD COLUMN last_scan bigint(20) NOT NULL DEFAULT UNIX_TIMESTAMP();
```

Comment:
Use `bigint unsigned`. It is perfectly fine to omit the `(20)` (which would be different with `unsigned` anyway).

We will change to milliseconds later, right? If so, please remove the default value.